sdks/typescript/featurehub-javascript-client-sdk/app/middleware.ts

```diff
@@ -101,6 +101,46 @@ class BaggageHolder implements FeatureStateHolder {
   triggerListeners(feature: FeatureStateHolder): void {
     this.existing.triggerListeners(feature);
   }
+
+  get enabled(): boolean {
+    return this.isEnabled();
+  }
+
+  get exists(): boolean {
+    return this.existing.exists;
+  }
+
+  get flag(): boolean | undefined {
+    return this.getBoolean();
+  }
+
+  get key(): string | undefined {
+    return this.getKey();
+  }
+
+  get locked(): boolean | undefined {
+    return this.isLocked();
+  }
+
+  get num(): number | undefined {
+    return this.getNumber();
+  }
+
+  get rawJson(): string | undefined {
+    return this.getRawJson();
+  }
+
+  get text(): string | undefined {
```

Comment:
hmm, not sure text() makes sense - you mean string feature, right?

Reply:
can we do get str()

Reply:
yep, we can, i couldn't  think of a name